inference_scripts/ms2deepscore_analog_discovery.py

```
@@ -229,8 +238,10 @@
 preprocessed_all_spectrums_janssen = all_spectrums_janssen.copy()
 if PREPROCESS_SPECTRUMS:
     for i, s in tqdm(enumerate(preprocessed_all_spectrums_janssen)):
```

Ruff (0.14.5)

Loop control variable `s` not used within loop body

(B007)